docs/planning/bug_triage.md

```diff
@@ -3,28 +3,30 @@
 This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
 This is a supplement to [Sprint planning](sprintplanning.md) process.
 
-## Goal
+## Goals
 
-- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
-- Process shouldn't be taxing for a single person or the team.
 - Maintain high product quality.
+- Keep a low bug count. Bug count should not be consistently increasing release over release especially [`priority/important-longterm`](#Bug-priorities).
+- Process shouldn't be taxing for a single person or the team.
 
 ## Bug priorities
 
-- priority/critical-urgent: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
-- priority/important-soon: Bugs that need to be fixed in the next release cycle, and but do not need a patch-fix.
-- priority/important-longterm: Bugs that need to be fixed, and will be fixed in next 2-3 releases.
-- priority/awaiting-more-evidence: Bugs which might need to be fixed, but need more details first.
+- `priority/critical-urgent`: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
+- `priority/important-soon`: Bugs that need to be fixed in the next release cycle, and but do not need a patch-fix.
+- `priority/important-longterm`: Bugs that need to be fixed, and will be fixed in next 2-3 releases.
+- `priority/awaiting-more-evidence`: Bugs which might need to be fixed, but need more details first. Bugs which remain open in awaiting-more-evidence may be closed (time out).
 
 ## How to manage bugs?
 
-- Submit new bugs anytime by creating a new issue of type “Bug”.
-- Please fill the bug template and optionally propose a priority.
-- All new bugs will be labelled “kind/bug” as part of the template
- Bugs will be triaged weekly (this could be modified based on incoming bugs
```

Comment:
I'm pretty sure this had lost the `<LI>` item. The others were some quoting fixes and clarification that the release target for bugs was tracked using milestones.